modules/olm-policy-scoping-operator-install.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-creating-policy.adoc
+
+[id="olm-policy-scoping-operator-install_{context}"]
+= Scoping Operator installations
+
+To provide scoping rules to Operator installations and upgrades on OLM,
+associate a service account with an OperatorGroup.
+
+Using this example, a cluster administrator can confine a set of Operators to a
+designated namespace.
+
+.Procedure
+
+. Create a new namespace:
++
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: scoped
+EOF
+----
+
+. Allocate permissions that you want the Operator(s) to be confined to. This
+involves creating a new service account, relevant Role(s), and RoleBinding(s).
++
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: scoped
+  namespace: scoped
+EOF
+----
++
+This exercise grants the service account permissions to do anything in the
+designated namespace for simplicity. In a production environment, you should
+create a more fine-grained set of permissions:
++
+----
+$ cat <<EOF | oc create -f -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: scoped
+  namespace: scoped
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: scoped-bindings
+  namespace: scoped
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: scoped
+subjects:
+- kind: ServiceAccount
+  name: scoped
+  namespace: scoped
+EOF
+----
+
+. Create an OperatorGroup in the designated namespace. This OperatorGroup targets
+the designated namespace to ensure that its tenancy is confined to it. In
+addition, OperatorGroups allow a user to specify a service account. Specify the
+ServiceAccount created in the previous step:
++
+----
+$ cat <<EOF | oc create -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: scoped
+  namespace: scoped
+spec:
+  serviceAccountName: scoped
+  targetNamespaces:
+  - scoped
+EOF
+----
++
+Any Operator being installed in the designated namespace is now be tied to this
+OperatorGroup and therefore to the service account specified.
+
+. Create a Subscription in the designated namespace to install an Operator. You
+can specify a CatalogSource that already exists in the designated namespace, or
+one that is in the global catalog namespace:
++
+----
+$ cat <<EOF | oc create -f -
+apiVersion: operators.coreos.com/v1
+kind: Subscription
+metadata:
+  name: etcd
+  namespace: scoped
+spec:
+  channel: singlenamespace-alpha
+  name: etcd
+  source: operatorhubio-catalog
```

Comment:
@tkashem 	@ecordell 	What would be a good example CatalogSource to show here? Should a step be added before this showing adding a custom one?

Reply:
@adellape `operatorhubio-catalog` ships in upstream olm installation. You can replace this with a placeholder name like `my-catalog-source-name`. I don't think we need to add steps that show how you create a CatalogSource. But if you have any openshift doc on how to create CatalogSource you can link it here maybe?